Applications/Subgraph/jni/meson.build

```diff
@@ -0,0 +1,11 @@
+subgraph_source = ['main.cpp', cifar_path / 'cifar_dataloader.cpp']
```

Comment:
same question : It seems that you are not using a data loader, is this necessary?